### PR TITLE
Limit comments

### PR DESCRIPTION
**Pull request by Ericas14**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -40,15 +41,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
-      for(Entity entity:results.asIterable() ){
+      for(Entity entity:results.asIterable(FetchOptions.Builder.withLimit(max)) ){
           String comment = (String)entity.getProperty("Comment");
           comments.add(comment);
       }
     response.setContentType("application/json"); 
     Gson gson = new Gson();
     response.getWriter().println(gson.toJson(comments));
+
+    //String commentMax = request.getParameter("loadComments");
```
Comment (jbandlow-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:56]: Please delete this.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -29,14 +29,34 @@ function addRandomFact() {
...
+      console.assert(commentContainer, "You have no element with ID 'comment-container'"); 
+
+      const list = document.getElementById('comment-container');
+      comment = comment.replace("[", "");
+      comment = comment.replace("]","");
+      var commentArray = comment.split(',');
+      for(i =0;  i < commentArray.length; i++){
+          list.appendChild(createListElement(commentArray[i]));
+        }
+    });
```
Comment (jbandlow-google) [inline portfolio/src/main/webapp/script.js:50]: It looks like some of this code is already under discussion in #14 . Please resolve those comments and merge those in and then ask for further review here.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -48,13 +48,21 @@ <h2>Academic Achievements</h2>
...
     <form action="data" method="POST">
         <input type="text" name="user-comment">
         <input type="submit"/>
     </form>
+    <br>
+    <form>
+    <p> Number of comments to show</p>
+    <input type = "number" name="load-comments" value ="1" max="10"> 
+    <input type = "submit">
+    </form>
```
Comment (mkruskal-google) [inline portfolio/src/main/webapp/index.html:62]: This works, but it requires a full page refresh to update the comments (and a button press from the user).  A less expensive method would be to add an onchange event handler to the number input element.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -48,13 +48,19 @@ <h2>Academic Achievements</h2>
...
     <br>
+    <h3>Give feedback!</h3>
     <form action="data" method="POST">
         <input type="text" name="user-comment">
         <input type="submit"/>
     </form>
+    <br>
+    <form>
+    <p> Number of comments to show</p>
+    <input type = "number" id="load-comments" onchange="getFromServlet()" value ="0" max="10" min="0">
```
Comment (mkruskal-google) [inline portfolio/src/main/webapp/index.html:60]: FYI, the "this" keyword actually contains the event within the onchange string.  So something like "getFromServlet(this.value)" would pass the value to the method without the method having to know which element it came from.

**Pull request merged**